Source/WebCore/workers/service/ServiceWorkerRegistration.cpp

```diff
@@ -311,7 +311,21 @@ void ServiceWorkerRegistration::showNotification(ScriptExecutionContext& context
         return;
     }
 
-    if (auto* serviceWorkerGlobalScope = dynamicDowncast<ServiceWorkerGlobalScope>(context)) {
+    auto* serviceWorkerGlobalScope = dynamicDowncast<ServiceWorkerGlobalScope>(context);
```

Comment:
RefPtr

Reply:
Definitely less sure about lifecycle for this one - Added.